### ISSUE TITLE
Behavior change

### ISSUE THREAD

**Issue by taynaud** [LuminosoInsight/python-ftfy#66]
Hello,

In last version, some examples are not working anymore

`
print(fix_text('\001\033[36;44mI&#x92;m blue, da ba dee da doo&#133;\033[0m', normalization='NFKC'))
`

gives `I'm blue, da ba dee da ba doo` and not `I'm blue, da ba dee da ba doo...`

```

Python 3.5.2 (default, Nov 17 2016, 17:05:23) 
[GCC 5.4.0 20160609] on linux
In[2]: from ftfy import fix_text
In[3]: print(fix_text('\001\033[36;44mI&#x92;m blue, da ba dee da ba doo&#133;\033[0m', normalization='NFKC'))
I'm blue, da ba dee da ba doo

In[4]: import ftfy
In[5]: ftfy.__version__
Out[5]: '5.0'

```

**Comment by rspeer** [comment]
Ah yes. I should have changed this example in the README like I did in the docs.

This example involving of `&#133;` (the HTML escaping of a wrongly encoded codepoint that was supposed to be an ellipsis, or the HTML escaping of the NEXT LINE control character) was a hypothetical example I came up with for the first version of ftfy, and its result depends on the order that fixes are applied in.

I changed the order of fixes to better handle real-world examples that I saw, at the expense of this hypothetical example.

I'll push a new version that updates the README, and describes this change better in the docs.

**Comment by rspeer** [comment]
Actually, no, you're right to report this as a bug. The old behavior was more consistent with what Web browsers do.

I'll add a special case for entities between `&#128;` and `&#159;`.

**Comment by rspeer** [comment]
Fixed in version 5.0.1.

**Comment by taynaud** [comment]
Thanks for the quick fix
